app/helpers/api/bapi/getCatalogueItems.test.js

```diff
@@ -0,0 +1,25 @@
+import { getData } from 'buying-catalogue-library';
+import { getCatalogueItems } from './getCatalogueItems';
+import { solutionsApiUrl } from '../../../config';
+import { logger } from '../../../logger';
+
+jest.mock('buying-catalogue-library');
+
+describe('getCatalogueItems', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('should call getData once with the correct params', async () => {
+    getData
+      .mockResolvedValueOnce({ data: {} });
+
+    await getCatalogueItems({ supplierId: 'supp-1', catalogueItemType: 'Solution', accessToken: 'access_token' });
+    expect(getData.mock.calls.length).toEqual(1);
+    expect(getData).toHaveBeenCalledWith({
+      endpoint: `${solutionsApiUrl}/api/v1/catalogue-items?supplierId=supp-1&catalogueItemType=Solution`,
+      accessToken: 'access_token',
+      logger,
+    });
```

Comment:
Need a test to check the response data or not?

Reply:
yeah I'll add one 👍